cypress/support/step-definitions/checkbox-steps.js

```diff
@@ -5,17 +5,14 @@ import {
 } from '../../locators/checkbox';
 import { label, fieldHelpPreview, getDataElementByValueNoIframe } from '../../locators';
 import { ICON } from '../../locators/locators';
+import { positionOfElement } from '../helper';
 
 const CHECKBOX_HELP_TEXT_CLASS_PREFIX = 'carbon-checkbox__help-text--';
 const CHECKBOX_LABEL_CLASS_PREFIX = 'common-input__label--';
 
 const FIRST_CHECKBOX = 0;
```

Comment:
do we need this const here ? It is never used anymore

Reply:
fixed